templates/github/.github/workflows/scripts/before_script.sh.j2

```diff
@@ -36,6 +36,14 @@ if [[ "${REDIS_DISABLED:-false}" == true ]]; then
   echo "The Redis service was disabled for $TEST"
 fi
 
+if [[ "{{ ci_base_image }}" = */pulp-ci-centos ]]; then
+  # centos-stream-8 was archived; start using vault.centos.org as a mirror when downloading packages
+  cmd_prefix bash -c "cd /etc/yum.repos.d/ \
```

Comment:
I think you don't need this line since `sed` is going directly to the files itself.